Packages/tests/UTF_HelperFunctions.ipf

```diff
@@ -781,3 +781,60 @@ Function/S GetTestStimsetFullFilePath()
 
 	return fullPath
 End
+
+Function/S LoadSweeps(string winAB)
+
+	PGC_SetAndActivateControl(winAB, "button_load_sweeps")
+
+	return StringFromList(0, WinList("*", ";", "WIN:" + num2istr(WINTYPE_GRAPH)))
+End
+
+/// @brief Open the given files, located relative to the symbolic path `home`, into an analysis browser
+Function [string abWin, string sweepBrowsers] OpenAnalysisBrowser(WAVE/T files, [variable loadSweeps])
+
+	variable idx
+	string filePath
+
+	if(ParamIsDefault(loadSweeps))
+		loadSweeps = 0
+	else
+		loadSweeps = !!loadSweeps
+	endif
+
+	PathInfo home
+	REQUIRE_EQUAL_VAR(V_flag, 1)
+
+	Duplicate/FREE/T files, filesWithPath
+	filesWithPath[] = S_path + files[p]
+
+	abWin = AB_OpenAnalysisBrowser(restoreSettings = 0)
+	CHECK_PROPER_STR(abWin)
+
+	for(file : filesWithPath)
```

Comment:
Missing "string file".

I would also suggest a different naming here, because we have "files" and "filesWithPath".
How about "fullFilePath" ?